### PR TITLE
Add organizations close_account feature

### PR DESCRIPTION
**Pull request by donelupus**

### CODE REVIEW

**Review by bblommers — CHANGES_REQUESTED** [review]
Hi @donelupus, thanks for the PR! Looks good overall, just got two minor comments

File: moto/organizations/models.py [hunk 1]
```diff
@@ -95,6 +95,19 @@ def create_account_status(self):
...
 
+    @property
+    def close_account_status(self):
+        return {
+            "CloseAccountStatus": {
+                "Id": self.create_account_status_id,
+                "AccountName": self.name,
+                "State": "SUCCEEDED",
+                "RequestedTimestamp": unix_time(self.create_time),
+                "CompletedTimestamp": unix_time(self.create_time),
```
[the remark this answers is not in the document]
Comment (donelupus) [reply moto/organizations/models.py]: Good point. I changed it accordingly.

File: moto/organizations/models.py [hunk 2]
```diff
@@ -445,6 +458,17 @@ def create_account(self, **kwargs):
...
+    def close_account(self, **kwargs):
+        for account_index in range(len(self.accounts)):
+            if kwargs["AccountId"] == self.accounts[account_index].id:
+                closed_account_status = self.accounts[
+                    account_index
+                ].close_account_status
+                del self.accounts[account_index]
+                return closed_account_status
+
+        raise AccountNotFoundException
```
[the remark this answers is not in the document]
Comment (donelupus) [reply moto/organizations/models.py:470]: I added one more test to check that the AccountNotFoundException is thrown.

**Review by bblommers — APPROVED** [review]
LGTM! Thanks for adding this feature @donelupus :+1:

**Pull request merged**